doc/fundamentals/append_overwrite.rst

```diff
@@ -1,571 +1,364 @@
+| `--target-file-datums` | This flag must be used with the `--split` <br> flag. The `--target-file-datums` attempts to fill each split-file <br>with the specified number of datums. If you run `--split lines --target-file-datums 2` on the line-delimited 50-line file mentioned above, you see the file split into 50 split-files and each file has 2 lines. |
+
+
+If you specify both `--target-file-datums` and `--target-file-bytes` flags,
+Pachyderm creates split-files until it hits one of the
+constraints. Pachyderm splits the file and
+then fills the first target split-file with line-based records
+until it hits the record limit. If Pachyderm passes the target byte
+number with just one record, it moves to the next split-file. If Pachyderm
+hits the target datum number after adding another line, it moves to the
+next split-file. If you run `pachctl put file` with
+`--split lines --target-file-datums 2 --target-file-bytes 100` flags
+on the example file in the table above, you get the same result as
+`--target-file-datums 2`, since that is the most compact constraint,
+and file sizes hover around 40 bytes.
+
+**See also:**
 
-.. important:: The ``list file`` command indicates that the line-oriented file we uploaded, “my-data.txt”, is actually a directory.
+- `Splitting Data for Distributed Processing <../cookbook/splitting.html#pg-dump-sql-support>`__
 
-This file *looks* like a directory because
-the ``--split`` flag has instructed Pachyderm to split the file up, and
-it has created a directory with all the chunks in it. And, as you can
-see below, each chunk will be put into a file. Those are the
-split-files. Each split-file will be given a 16-character filename,
-left-padded with 0.
+### Example: Splitting a File
 
-.. note:: ``--split`` does not currently allow you to define more sophisticated file names.
-         This is a set of features we'll add in future releases.
-         (See `Issue 3568 <https://github.com/pachyderm/pachyderm/issues/3568>`__, for example).
+In this example, you have a 50-line file called `my-data.txt`.
+You create a repository named `line-data` and load
+`my-data.txt` into that repository. Then, you can analyze
+how the data is split in the repository.
 
-Each filename will be numbered sequentially in hexadecimal. We
-modify the command to list the contents of “my-data.txt”, and the output
-reveals the naming structure used:
+To complete this example, perform the following steps:
 
-::
+1. Create a file with fifty lines named `my-data.txt`. You can
+   add random lines, such as numbers from one to fifty, or US states,
+   or anything else..
+
+   **Examples:**
+
+   ```bash
```

Comment:
minor nit. For any counting examples like this, you should start the file with "zero" not one. That way file 0001 has contents "one" instead of "two". This applies to both examples here.